databox/api/src/Workspace/WorkspaceTemplater.php

```diff
@@ -0,0 +1,394 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Workspace;
+
+use App\Entity\Core\AttributeClass;
+use App\Entity\Core\AttributeDefinition;
+use App\Entity\Core\RenditionClass;
+use App\Entity\Core\RenditionDefinition;
+use App\Entity\Core\Tag;
+use App\Entity\Core\Workspace;
+use Doctrine\ORM\EntityManagerInterface;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\String\Slugger\AsciiSlugger;
+
+final readonly class WorkspaceTemplater
+{
+    public function __construct(
+        private EntityManagerInterface $em,
+        private LoggerInterface $logger,
+    ) {
+    }
+
+    public function export(Workspace $workspace): string
+    {
+        return json_encode([
+            'Workspace' => $this->exportWorkspace($workspace),
+            'RenditionClass' => $this->exportRenditionClass($workspace->getId()),
+            'RenditionDefinition' => $this->exportRenditionDefinition($workspace->getId()),
+            'AttributeClass' => $this->exportAttributeClass($workspace->getId()),
+            'AttributeDefinition' => $this->exportAttributeDefinition($workspace->getId()),
+            'Tag' => $this->exportTag($workspace->getId()),
+        ], JSON_PRETTY_PRINT);
+    }
+
+    public function import(string $data, string $newName, string $ownerId): void
+    {
+        $data = json_decode($data, true);
+
+        $slugger = new AsciiSlugger();
+        $this->em->beginTransaction();
+        try {
+            /** @var Workspace $ws */
+            if (!($ws = $this->em->getRepository(Workspace::class)->findOneBy(['name' => $newName]))) {
+                $this->logger->info(sprintf('Creating Workspace "%s"', $newName));
+                $ws = new Workspace();
+                $ws->setOwnerId($ownerId);
+                $ws->setName($newName);
+                $ws->setSlug($slugger->slug($newName)->toString());
+            } else {
+                $this->logger->info(sprintf('Updating Workspace "%s"', $newName));
```

Comment:
We should add a security here, an option passed via the command to ensure the user is aware that an existing workspace will be modified.